language/English/strings.po

```diff
@@ -11579,7 +11579,43 @@ msgctxt "#21470"
 msgid "%s [%d]"
 msgstr ""
 
-#empty strings from id 21471 to 21601
+#: system/settings/settings.xml
+msgctxt "#21471"
+msgid "Initially"
+msgstr ""
```

Comment:
there should be descriptions added for translators (for all new strings preferably)